Makefile

```diff
@@ -26,9 +30,23 @@ check-license:
 
 tools: $(TOOLS)
 
+man: $(MAN)
+
+all: $(TOOLS) $(MAN)
+
 $(TOOLS): oci-%:
 	go build -ldflags "-X main.gitCommit=${COMMIT}" ./cmd/$@
 
+$(MAN): %.1: cmd/*/*.1.md
```

Comment:
> it's ok

It's not ok, it's using `oci-create-runtime-bundle.1.md` as the source for all the pages.

> it will be hang

That's because you left off the `SECONDEXPANSION:` line.  It's important ;).  Doc link in my original secondary-expansion comment.